plugins/modules/cluster_role_info.py

```diff
@@ -70,9 +70,10 @@ def main():
         ),
```

Comment:
Please also remove _Parameter C(auth.namespace) is ignored in this module_ from the docs for this module.